fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/variableinstallments/VariableInstallmentsFlatHelper.java

```diff
@@ -201,6 +201,7 @@ private static ArrayList createModifyMap(String date) {
         return toReturn;
     }
 
+    @SuppressWarnings("MisusedWeekYear")
```

Comment:
@vorburger not sure the of the right format that was supposed to be on methods I added @SuppressWarnings("MisuedWeekYear") please see https://errorprone.info/bugpattern/MisusedWeekYear

Reply:
this seems suspicious - are you sure suppressing this is correct, and not actually hiding a real bug? (I haven't had a closer look, I'm hoping you'll do that...) If you are completely sure, then each `@SuppressWarnings` should have an inline comment with a reasonably detailed explanation why that is the right thing to do.